SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WriteLinuxScript/WriteLinuxScript.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why does this line appear to be changed in every csproj file

Reply:
No idea... maybe the TextEditor on Mac updated the file header?